README.md

```diff
@@ -1,7 +1,64 @@
 # CommonFactory
- A description of what is it and what it is used for.  
-  
-  ## Usage 
-  How to use the CommonFactory script
-  ## APIs
-  - APIs list
+Package with helper scripts to deploy a Common DAO using Arc `DAOFactory`.
+
+# Usage
+Run: `npm install --save @daostack/commonfactory`
+
+Import the `commonfactory` methods: `getForgeOrgData, getSetSchemesData` and use them to generate call data to Arc `DAOFactory` methods.
+```javascript
+const { getForgeOrgData, getSetSchemesData } = require('@daostack/commonfactory');
+
+// ...
+const forgeOrg = await daoFactory.methods.forgeOrg(
+      ...getForgeOrgData({
+        DAOFactoryInstance,
+        orgName,
+        founderAddresses: [web3.eth.accounts.wallet[0].address],
+        tokenDist: [0],
+        repDist: [100]
+      })
+    ).send();
+
+// ...
+const setSchemes = await daoFactory.methods.setSchemes(
+      ...getSetSchemesData({
+        DAOFactoryInstance,
+        avatar: avatarAddress,
+        votingMachine,
+        joinAndQuitVoteParams: "0x1000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
wait what? These values I'd like to generate with the script, not provide TO the script

Reply:
If they are the same for all commons, it will be possible, but if not then I can't generate them myself.

Reply:
I guess they will be the same for all common's per scheme.

Reply:
If so then when we have those hashes I can make them hardcoded.